lib/ui/painting/vertices.cc

```diff
@@ -44,48 +27,48 @@ void Vertices::RegisterNatives(tonic::DartLibraryNatives* natives) {
 }
 
 bool Vertices::init(Dart_Handle vertices_handle,
-                    SkVertices::VertexMode vertex_mode,
+                    DlVertexMode vertex_mode,
                     tonic::Float32List& positions,
                     tonic::Float32List& texture_coordinates,
                     tonic::Int32List& colors,
                     tonic::Uint16List& indices) {
   UIDartState::ThrowIfUIOperationsProhibited();
-  uint32_t builderFlags = 0;
+
+  if (!positions.data()) {
+    return false;
```

Comment:
@dnfield Does the type data need to be `Release()ed` before returning?

Looks like it's this test failing: https://github.com/flutter/engine/blob/main/testing/dart/canvas_test.dart#L99

Reply:
Hm. I would have thought that it would get Releas()ed on collection here. There should not be any other Dart API getting called doing allocations. But it's worth a shot adding it.

But in that case we'd probably need to release _all_ of the typed lists here right...?

We perhaps need some explicit releases below on L56

Reply:
The crash is in this function https://github.com/dart-lang/sdk/blob/main/runtime/vm/dart_api_impl.cc#L5377. So something is not in a happy state on return, and I can't really figure what else it might be.

Reply:
ahhh! Yes.

haha this does an allocation in tonic, becuase it's allocating a false object. We have to release all the typed data that was passed in before returning here.

Reply:
The crash is due to me rejecting vertex_count == 0 and isn't due to not releasing those arrays, but probably becomes a source of failures to release.

Note that these "return false" without releasing chunks of code have always been there. I don't see any calls to "Release" in canvas.cc which has a lot of tonic typed lists...?